.github/workflows/4-deploy-to-prod.yml

```diff
@@ -0,0 +1,91 @@
+name: Deploy to prod
+on:
+  push:
+    branches:
+      - 'releases/prod'
+
+jobs:
+  deploy-to-prod:
+    environment:
+      name: prod
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+      - run: yarn prepare
```

Comment:
`yarn prepare` is now broken into its components in `actions/setup` to better utilize caching, but I think it should remain here so that the build is not dependent on caching.